aeolis/console.py

```diff
@@ -13,31 +14,88 @@
 You should have received a copy of the GNU General Public License
 along with AeoLiS.  If not, see <http://www.gnu.org/licenses/>.
 
-AeoLiS  Copyright (C) 2023 AeoLiS Development Team
+AeoLiS  Copyright (C) 2015 Bas Hoonhout
+
+bas.hoonhout@deltares.nl         b.m.hoonhout@tudelft.nl
+Deltares                         Delft University of Technology
+Unit of Hydraulic Engineering    Faculty of Civil Engineering and Geosciences
+Boussinesqweg 1                  Stevinweg 1
+2629 HVDelft                     2628CN Delft
+The Netherlands                  The Netherlands
+
+
+***************************************************************************
```

Comment:
I discussed the IP issues with Sierd and he said that Bas was ok with doing whatever they wanted with the original source code. Sierd said he preferred to mention the AeoLiS team. if you discussed this with Sierd, please ignore these comments.

Reply:
Thanks, I remember we discussed it already before. This is a typo on my end while trying to update the docstring. I will restore the old copyright text that we had which mentioned Aeolis development team.